doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -149,6 +149,22 @@ in case you set this option will have the priority compared to the one defined i
 |``
 |Yes^*^
 
+|`relationship.node.map`
```

Comment:
I like this setting, would suggest to flatten the DF by default if this is feasible, because it prevents the user from having to destructure it and it's more "relational friendly", making the map behavior something you can toggle instead.

I find the name a bit confusing.  "Relationship node?" what's that.  :) 

Not totally clear on what's being said here.  A relationship is a property map but so are the nodes.  Is this setting flattening *all* of them, or just the relationship one?    Maybe a more clear name would be properties.flatten=True

This option doesn't appear to work for me:

```
%py

df = spark.read.format("org.neo4j.spark.DataSource") \
  .option("url", dbutils.widgets.get("url")) \
  .option("authentication.basic.username", dbutils.widgets.get("user")) \
  .option("authentication.basic.password", dbutils.widgets.get("password")) \
  .option("relationship", "ORG") \
  .option("relationship.source.labels", "Event") \
  .option("relationship.node.map", False) \
  .option("relationship.target.labels", "Organization").load()

df.printSchema()
```

yields

```
root
 |-- <id>: long (nullable = false)
 |-- <relationshipType>: string (nullable = false)
 |-- <source>: map (nullable = false)
 |    |-- key: string
 |    |-- value: string (valueContainsNull = true)
 |-- <target>: map (nullable = false)
 |    |-- key: string
 |    |-- value: string (valueContainsNull = true)
```

Reply:
`relationship.nodes.as.map` could be a better naming?

Reply:
When you use both the word relationship and node in the same setting, that's what's confusing me, because at once I can't tell if you're referring to one, the other, or both.   I guess I would think that "relationship.nodes.as.map" would return *node* properties as a map.  But then I'd be left wondering what happens to relationship properties.  (But they follow the same setting!)